testproj/testproj/settings.py

```diff
@@ -126,3 +126,6 @@
 WEBAUTHN_TRUSTED_CERTIFICATES = os.path.join(
     BASE_DIR, "..", "trusted_attestation_roots"
 )
+WEBAUTHN_TRUSTED_ATTESTATION_CERT_REQUIRED = True
```

Comment:
Repeating the same comment from #18... Based on the [related discussion](https://github.com/solokeys/solo/issues/275#issuecomment-522170055) about potential concerns with whitelisting attestation certs, I get the impression it might be better to set the default value for this setting to False. What do you think?

Reply:
Yes I fixed it in #18

Reply:
Sure, but doesn't it need to be changed here in the demo project as well?

Reply:
If we plan on using default settings in the demo project I am going to comment those lines once we merge #18.